### PR TITLE
chore(css): Fix various css warnings and adjust margins for about:welcome

### PR DESCRIPTION
**Pull request by sarracini**
a variety of things that were annoying me showing up as warnings
63 problems down to 35
exciting times

### CODE REVIEW

File: content-src/components/StartupOverlay/_StartupOverlay.scss [hunk 1]
```diff
@@ -100,7 +100,7 @@
   .extra-links {
     font-size: 12px;
     max-width: 340px;
-    margin: 14px 50px;
+    margin: 17px 50px;
```
Comment (sarracini) [inline content-src/components/StartupOverlay/_StartupOverlay.scss]: erica asked me to sneak this change, and the margin change in `.skip-button` into this pr while i'm here

File: content-src/styles/_variables.scss [hunk 2]
```diff
@@ -51,6 +51,8 @@ $pocket-teal: #50BCB6;
 $bookmark-icon-fill: #0A84FF;
 $download-icon-fill: #12BC00;
 $pocket-icon-fill: #D70022;
+$email-input-focus: rgba(10, 132, 255, 0.3);
```
Comment (piatra) [inline content-src/styles/_variables.scss]: This is already defined as $blue-50.

File: content-src/styles/_variables.scss [hunk 3]
```diff
@@ -51,6 +51,8 @@ $pocket-teal: #50BCB6;
...
+$email-input-invalid: rgba(215, 0, 34, 0.3);
```
Comment (piatra) [inline content-src/styles/_variables.scss]: This one is `$red-60`.

File: content-src/components/StartupOverlay/_StartupOverlay.scss [hunk 4]
```diff
@@ -299,15 +297,15 @@
...
   display: none;
-}
 
-html[dir="rtl"] .error{
-  right: 50px;
-  left: auto;
-  
-  &::before {
-    right: 12px;
+  &:dir(rtl) {
```
Comment (piatra) [inline content-src/components/StartupOverlay/_StartupOverlay.scss]: I don't think the rewrite means the same thing:
```
html[dir] .error
```
vs
```
.error:dir(rtl)
```

Comment (sarracini) [reply content-src/components/StartupOverlay/_StartupOverlay.scss]: i'll just change the left/right to use offset-inline-start and everything will sort itself out in rtl!

File: content-src/components/StartupOverlay/_StartupOverlay.scss (hunk 1 above)
Comment (k88hudson) [reply content-src/components/StartupOverlay/_StartupOverlay.scss]: can you add that to the commit message

**Comment by k88hudson** [comment]
HERO 🌟

**Pull request merged**